pymeasure/instruments/newport/esp300.py

```
@@ -253,10 +252,26 @@ def __init__(self, adapter, name="Newport ESP 300 Motion Controller", **kwargs):
         )
-        # Defines default axes, which can be overwritten
-        self.x = Axis(1, self)
-        self.y = Axis(2, self)
-        self.phi = Axis(3, self)
+        self.axes = [Axis(1, self), Axis(2, self), Axis(3, self)]
+
+    @property
+    def x(self):
+        warn("The x attribute is deprecated."
+             "Axes are now directly accessed through the axes attribute.",
+               FutureWarning, stacklevel=2)
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)